packages/core/src/actions/fetchData.js

```diff
@@ -1,56 +1,79 @@
 import axios from "axios";
 import PromiseThrottle from "promise-throttle";
 import {encodeChars} from "../helpers/fixChars.js";
+import "@babel/polyfill";
```

Comment:
`"@babel/polyfill"` is actually included as an entry point in the prod and dev client webpack configs... maybe it just needs to also be added to the server webpack configs instead of this import? I just get nervous importing this in two places (potential code duplication, etc)

Reply:
Good one. I added to webpack, in server build. Looks better.